### PR TITLE
Making dmwmbld script recognize the status.

### PR DESCRIPTION
**Pull request by arooshap**

### CODE REVIEW

File: dmwmbld/gh-bot [hunk 1]
```diff
@@ -26,13 +26,18 @@ def generate_index_page(pr, status, cfgdir, statedir):
 # Test builds for pull requests against an specific branch in the gh repo.
 # Update the pull req status according to overall result of tests on all
 # the architectures.
+def check_context(gh, pull_request, owner, repo, branch):
```
Comment (smuzaffar) [inline dmwmbld/gh-bot:29]: @arooshap , looks good overall.
Just a minor/last fix: `owner, repo, branch` are not used in `check_context` function, so I would suggest to remove these and update the call `check_context(gh, i, owner, repo, branch)` to `check_context(gh, i)`

File: dmwmbld/gh-bot [hunk 2]
```diff
@@ -26,13 +26,18 @@ def generate_index_page(pr, status, cfgdir, statedir):
...
+    # Check if there's a status with context equal to "cms/bot"
+    return any(status['state'] == 'success' and status.get('context') == 'cmsbot/test' for status in statuses)
 def pull_req_tester(gh, owner, repo, branch, arch, all_archs, statedir, logdir, logurl, cfgdir, authfile):
   # get the relevant pull requests
   valid_comp_users = json.load(open(usersfile))
   PRs = [i for i in gh.GetPullRequests(owner, repo) \
-           if (((i['base']['label'] == owner + ':' + branch) and
-           (i['user']['login'].encode('ascii', 'ignore') in valid_comp_users)) or
-           (any(label['name'] == 'cmsbot/test' for label in i.get_labels())))]
+        if (i['base']['ref'] == 'comp_gcc630' and ((i['user']['login'].encode('ascii', 'ignore') in valid_comp_users) or check_context(gh, i, owner, repo, branch)))]
```
Comment (smuzaffar) [inline dmwmbld/gh-bot:40]: also can we avoid hard coded `comp_gcc630` here? shouldn't it be `branch` instead of `comp_gcc630`

Comment (arooshap) [reply dmwmbld/gh-bot:40]: Oh, I thought you wanted it like this when you asked me to add this check. I will make the changes. Thank you!

**Comment by smuzaffar** [comment]
looks good. feel free to merge it

**Pull request merged**